app/controllers/ProjectSidewalkAPIController.scala

```diff
@@ -323,8 +428,16 @@ class ProjectSidewalkAPIController @Inject()(implicit val env: Environment[User,
         globalAttributes.map(l => AttributeForAccessScore(l.lat, l.lng, l.labelType))
     }
     val streetEdges: List[StreetEdge] = StreetEdgeTable.selectAuditedStreetsWithin(minLat, minLng, maxLat, maxLng)
-    val streetAccessScores = computeAccessScoresForStreets(streetEdges, labelsForScore)
+    computeAccessScoresForStreets(streetEdges, labelsForScore) 
+  }
 
+  /**
+    * Generic version of getAccessScoreStreets, specifically when the client wants the data in CSV format
+    *
+    * @param streetAccessScores         list of AccessScoreStreet objects for specified region
+    * @param return
+    */
+  def getGeoJSONAccessScoreStreetsGeneric(streetAccessScores: List[AccessScoreStreet]): JsObject  = {
```

Comment:
You should just turn this into a `toJSON` method of the `AccessScoreStreet` class. Then you can call `.map(_.toJSON)` the same way we do elsewhere in this file to be more consistent.